apps/example/e2e/README.md

```diff
@@ -22,35 +20,37 @@ Install [AppleSimulatorUtils](https://github.com/wix/AppleSimulatorUtils#install
 
 ### Setting up secrets for local runs
 
-An `.env` example is present at `e2e/env.example` rename this file to `.env` and populate the secrets from Google Cloud Platform. Do not commit the newly created `e2e/.env` file.
+An `.env` example is present at `apps/example/e2e/.env` rename this file to `.env` and populate the secrets from Google Cloud Platform. Do not commit the newly created `apps/example/e2e/.env` file.
 
 ## Running the tests
 
 ```sh
-# Create your detox build various options present in the package.json
+# (Android Only) - Run the prebuild command to include Detox in the build
+yarn e2e:prebuild
+
 # Only needs to be built when app code changes - test code can be changed without a new build.
-yarn run e2e:build:android-release
+yarn e2e:build:android-release
 
 # (optional) - When running the e2e tests on a debug build the e2e packager should be run in a separate terminal first.
-yarn run e2e:packager
+yarn e2e:packager
 
-# Run Detox
+# Run the entire Detox test suite
 yarn run e2e:test:android-release
 ```
 
```

Comment:
I wonder if we should document something around the test butler apk?

Reply:
I am able to run the e2e tests with the test Butler app enabled. If this continues to be an issue, I am happy to take another look.